common/src/main/java/org/opensearch/ml/common/connector/AbstractConnector.java

```diff
@@ -43,6 +43,8 @@ public abstract class AbstractConnector implements Connector {
     public static final String BACKEND_ROLES_FIELD = "backend_roles";
     public static final String OWNER_FIELD = "owner";
     public static final String ACCESS_FIELD = "access";
+    public static final String HTTP_CLIENT_CONFIG_FIELD = "http_client_config";
```

Comment:
Should we just name it as `config` ?  We may support other protocol in future which is not http

Reply:
renamed it to `client_config`